src/components/PopupWithForm.js

```diff
@@ -30,19 +40,13 @@ class PopupWithForm extends Popup {
 
   open() {
     super.open();
-    this._button.textContent = "Save";
+    this._popupForm.reset();
   }
 
   close() {
-    this._popupForm.reset();
```

Comment:
`this._popupForm.reset();` should be in `close` according to the task of the 8th sprint